src/e2e/resources/simple-temple-expected/simple-temple-test-group/simple-temple-test-group.go

```diff
@@ -189,7 +189,75 @@ func (env *env) readSimpleTempleTestGroupHandler(w http.ResponseWriter, r *http.
 }
 
 func (env *env) updateSimpleTempleTestGroupHandler(w http.ResponseWriter, r *http.Request) {
+	auth, err := util.ExtractAuthIDFromRequest(r.Header)
+	if err != nil {
+		errMsg := util.CreateErrorJSON(fmt.Sprintf("Could not authorize request: %s", err.Error()))
+		http.Error(w, errMsg, http.StatusUnauthorized)
+		return
+	}
 
+	simpleTempleTestGroupID, err := util.ExtractIDFromRequest(mux.Vars(r))
+	if err != nil {
+		http.Error(w, util.CreateErrorJSON(err.Error()), http.StatusBadRequest)
+		return
+	}
+
+	authorized, err := checkAuthorization(env, simpleTempleTestGroupID, auth)
+	if err != nil {
+		switch err.(type) {
+		case dao.ErrSimpleTempleTestGroupNotFound:
+			errMsg := util.CreateErrorJSON("Unauthorized")
+			http.Error(w, errMsg, http.StatusUnauthorized)
+		default:
+			errMsg := util.CreateErrorJSON(fmt.Sprintf("Something went wrong: %s", err.Error()))
+			http.Error(w, errMsg, http.StatusInternalServerError)
+		}
+		return
+	}
+	if !authorized {
+		errMsg := util.CreateErrorJSON("Unauthorized")
+		http.Error(w, errMsg, http.StatusUnauthorized)
+		return
+	}
+
+	var req updateSimpleTempleTestGroupRequest
+	err = json.NewDecoder(r.Body).Decode(&req)
+	if err != nil {
+		errMsg := util.CreateErrorJSON(fmt.Sprintf("Invalid request parameters: %s", err.Error()))
+		http.Error(w, errMsg, http.StatusBadRequest)
+		return
+	}
+
+	if {
```

Comment:
wat

Reply:
Ah this is part of the `Handle the case of no user-defined/client-facing attributes`, will be handled in another PR

Reply:
Can you at least make it valid Go? `if false`?